scipy/ndimage/_ni_support.py

```diff
@@ -95,3 +96,24 @@ def _get_output(output, input, shape=None, complex_output=False):
     elif complex_output and output.dtype.kind != 'c':
         raise RuntimeError("output must have complex dtype")
     return output
+
+
+def _check_axes(axes, ndim):
+    if axes is None:
+        return tuple(range(ndim))
+    elif isinstance(axes, Integral):
+        axes = (axes,)
+    if isinstance(axes, Iterable):
```

Comment:
I would have suggested `Sequence` since order matters, but apparently an `ndarray` isn't a `Sequence`, so I suppose this is fine.

Reply:
I would also be okay with being strict about only supporting `tuple`, but this way is more in line with how the rest of the arguments are currently handled in the `ndimage` module. (`_normalize_sequence` in this same file also uses `Iterable`)

Reply:
I agree. Maybe in SciPy 2.0 we can make this sort of thing strict, but I think for now we probably should be flexible.